### PR TITLE
Update 04-create-schema.md

### PR DESCRIPTION
**Pull request by MathewBiddle**
Addressing proposed changes in #41 

Cherry picking changes from #42

### CODE REVIEW

**Comment by MathewBiddle** [comment]
@ymgan do these changes look right?

https://github.com/ioos/bio_mobilization_workshop/pull/43/files

**Comment by ymgan** [comment]
Thanks Mathew! The changes look right! I spotted 2 more outdated links and commented at that line~
Thanks a lot for your hard work!

File: _episodes/04-create-schema.md [hunk 1]
```diff
@@ -31,12 +31,12 @@ The bare minimum for sharing data to OBIS is to use the [Occurrence Core](https:
...
+An innovation that OBIS made in this space was introducing the Extended Measurement or Fact extension (also sometimes referred to as OBIS-ENV-DATA). This uses the event core with an occurrence extension + the [extended measurement or fact extension](https://rs.gbif.org/extension/obis/extended_measurement_or_fact.xml). This extension makes it possible to include measurements for both the events (salinity, temperature, dissolved oxygen, gear type, etc.) as well as measurements about the occurrences (weight, length, etc.). Prior to this you were only able to include measurements of the occurrence.
 
-<img width="800" alt="EventCoreSchema" src="https://obis.org/images/EventCoreSchema.png" style="background-color:white;" />
+<img width="800" alt="EventCoreSchema" src="https://ipt.gbif.org/manual/en/ipt/latest/_images/figures/dwc-a_event.png" style="background-color:white;" />
```
Comment (MathewBiddle) [inline _episodes/04-create-schema.md:39]: @albenson-usgs is this the right image we want here? I don't recall what the previous image was, and the url was less than helpful.

Comment (MathewBiddle) [reply _episodes/04-create-schema.md:39]: We can fix later if wrong.

**Pull request merged**